web/pageComponents/shared/IframeCarousel/IframeCarousel.tsx

```diff
@@ -25,8 +26,16 @@ const Figure = styled.figure`
   width: 100%;
 `
 const ItemContainer = styled.div`
+  min-width: 45%;
```

Comment:
could also use `min-width: calc(50% - var(--space-medium))` (50% minus the gap of the grid)